### PR TITLE
chore: Update browser ajax data troubleshooting

### PR DESCRIPTION
**Pull request by patrickhousley**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

This PR updates the ajax missing data troubleshooting doc. The browser agent has supported capturing `fetch` for some time and has made strides in the data we expose on the window to help with troubleshooting agent initialization.

Also fixed a bug in the sidebar menu where click on the link for `page-load-timing-resources/instrumentation-browser-monitoring` would completely collapse the sidebar menu.

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx [hunk 1]
```diff
@@ -1,64 +1,81 @@
 ---
-title: Troubleshoot AJAX data collection
+title: Troubleshoot Ajax data collection
```
Comment (metal-messiah) [inline src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx:2]: `AJAX` is an acronym, so it's better displayed in all caps when trying to be "proper"

File: src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx [hunk 2]
```diff
@@ -1,64 +1,81 @@
...
-      If you are making requests using [JSONP](https://en.wikipedia.org/wiki/JSONP), see [JSONP requirements](#jsonp).
-  </Collapser>
-
   <Collapser
     id="verify-instrumentation"
-    title="2. Verify the object is instrumented."
+    title="1. Verify the page is instrumented."
   >
+    Use your browser's dev console to verify page has been correctly instrumented. Enter `XMLHttpRequest` and/or `fetch` into the console.
     If you are using `XMLHttpRequest`, use your browser's dev console to verify that the object has been instrumented by New Relic. Enter the object name at your console.
```
[the remark this answers is not in the document]
Comment (metal-messiah) [reply src/content/docs/browser/new-relic-browser/troubleshooting/troubleshoot-ajax-data-collection.mdx:27]:
```suggestion

```

**Comment by akristen** [comment]
Currently working with Eng to figure out why this isn't building, then we can merge this today!

**Comment by clarkmcadoo** [comment]
`Terminating the process (due to GATSBY_WATCHDOG_STUCK_STATUS_TIMEOUT):`

**Review by akristen — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**